versions/3.0.md

```diff
@@ -74,12 +74,13 @@ Additional utilities can also take advantage of the resulting files, such as tes
 ## Revision History
 
 Version | Date | Notes
---- | --- | ---
-2.0 | 2015-12-31 | Donation of Swagger 2.0 to the Open API Initiative
-2.0 | 2014-09-08 | Release of Swagger 2.0
-1.2 | 2014-03-14 | Initial release of the formal document.
-1.1 | 2012-08-22 | Release of Swagger 1.1
-1.0 | 2011-08-10 | First release of the Swagger Specification
+---   | --- | ---
+3.0.0 | 2017-02-28 | Implementor's draft of the 3.0 specification
```

Comment:
I suggest we follow a model of calling this v3.0.0-rc0 (release candidate zero) and then when we have polished, we're at v3.0.0-rc1 (or rc2 or rc3) and then we have a v3.0.0-final (and can delete the release candidates)